addons/storyshots/src/react/renderTree.js

```diff
@@ -2,6 +2,13 @@
 import reactTestRenderer from 'react-test-renderer';
 
 function getRenderedTree(story, context, { renderer, serializer, ...rendererOptions }) {
+  if (serializer) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'The "serializer" option of @storybook/addon-storyshots  has been deprecated. Please use "snapshotSerializers: [<your serializer>]" in the future.'
+    );
+  }
+
```

Comment:
so, we will need to change this line in the end: `return serializer ? serializer(tree) : tree;`, right ?

I would do some thing like this:

```js
if (serializer) {
  console.warn('blah blah');
  return serializer(tree);
}

return tree;
```

that way it will be clearer what to remove after the deprecation.